README.md

```diff
@@ -0,0 +1,70 @@
+# Rancher Labs Documentation
+This readme file includes information pertaining to the documentation project for both Rancher and Rancher OS.
+## Rancher
+Rancher is an open source project that provides a complete platform for operating Docker in production. It provides infrastructure services such as multi-host networking, global and local load balancing, and volume snapshots. It integrates native Docker management capabilities such as Docker Machine and Docker Swarm. It offers a rich user experience that enables devops admins to operate Docker in production at large scale.
+## Rancher OS
+The smallest, easiest way to run Docker in production at scale. Everything in RancherOS is a container managed by Docker. This includes system services such as udev and rsyslog. RancherOS includes only the bare minimum amount of software needed to run Docker. This keeps the binary download of RancherOS to about 20MB. Everything else can be pulled in dynamically through Docker.
+## Rancher Labs Documentation Website
+Rancher documentation is available at <http://docs.rancher.com/rancher/>
+Rancher OS documentation is available at <http://docs.rancher.com/os/>
+## Contributing to Rancher Labs Documentation Project
+### About Rancher Labs Documentation Site
+Rancher Lab documentation is hosted on Githubpages, and published it online by using Jekyll, an easy blog-aware static website generator. For more details on how to set up Jekyll, we recommend you to read <https://help.github.com/articles/using-jekyll-with-pages/>. If you are using Windows, we strongly advise you to follow the instruction given at <http://jekyllrb.com/docs/windows/>.
+For information on editing .md files (Markdown), refer to <https://daringfireball.net/projects/markdown/syntax>
+### Setting up the Git Environment
+In your browser, navigate to <https://github.com/rancher/rancher.github.io>
+Fork this repository by clicking the Fork button on the upper right-hand side. A fork of the repository is generated for you. On the right-hand side of the page of your fork, under 'HTTPS clone URL', copy the URL by clicking the Copy to clipboard icon.
+On your computer, follow these steps to setup a local repository to start working on the documentation:
+ 
+    $ git clone https://github.com/YOUR_ACCOUNT/rancher.github.io.git
+    $ cd rancher.github.io
+    $ git remote add upstream https://github.com/rancher/rancher.github.io
+    $ git checkout master
+    $ git fetch upstream
+    $ git merge upstream/master
+  
```

Comment:
For all your code blocks can you add "```" in front of it and at the bottom of them? It'll make them into fenced code blocks which are nice markdown. 

Also, code blocks in your text can be put in single backticks so that it looks different. `dev`.